src/aoscene.cpp

```diff
@@ -43,9 +43,12 @@ void AOScene::set_image(QString p_image)
 
     m_movie->setScaledSize(QSize(n_w, n_h));
     this->resize(m_movie->scaledSize());
-    this->setMovie(m_movie);
+    if (!file_exists(background_path) || background_path != last_image)
```

Comment:
suggest extracting variable `restart`

Reply:
@oldmud0 instructions unclear, rebased entire DRO codebase into AO2-Client

Reply:
seriously tho what do you mean lol

Reply:
the logic is a copypaste from the previous check